cmp/grid/GridModel.js

```diff
@@ -250,6 +252,10 @@ export class GridModel extends HoistModel {
      *      should not immediately respond to user or programmatic changes to the sortBy property,
      *      but will instead wait for the next load of data, which is assumed to be pre-sorted.
      *      Default false.
+     * @param {boolean} [c.highlightRowOnClick] - Set to true to highlight a row on click. Intended
+     *      to provide feedback to users in grids without selection. Note that setting this changes
+     *      the default color provided to Column.highlightOnChange. Default true for phones and
+     *      tablets, false for desktop.
```

Comment:
Maybe something like "Note this setting overrides the styling used by Column.highlightOnChange, and is not recommended for use alongside that feature."